docs/content/topics/operators/deploy.md

```diff
@@ -11,4 +11,429 @@ aliases:
+
+[Cert Manager]: https://cert-manager.io/docs/
+[ACME]: https://cert-manager.io/docs/configuration/acme/
+[Let's Encrypt]: https://letsencrypt.org/
+
+### Deploy Cert Manager
+
+To continue the theme of deploying these auxiliary services to Kubernetes via
+Helm, we'll install Cert Manager via its Helm chart:
+
+```console
+$ helm repo add jetstack https://charts.jetstack.io
+$ helm repo update
+$ helm install \
+  cert-manager jetstack/cert-manager \
+  --namespace cert-manager \
+  --create-namespace \
+  --version v1.4.0 \
+  --set installCRDs=true
+```
+
+_Note: This is where the Kubernetes 1.18.8+ and Helm v3.3.1+ constraints come
+in.  For more info, see [Cert Manager's upgrade notes]_
+
+[Cert Manager's upgrade notes]: https://cert-manager.io/docs/installation/upgrading/upgrading-0.15-0.16/#helm
+
+### Create Cert Issuers
+
+We now need to configure the [ACME Cert Issuer resource] and create it via
+`kubectl`.  We'll only create a production resource, but you may also create
+a staging resource in addition or instead.  See the Cert Manager docs for more
+info.
+
+The prod resource (to be saved as `cert-manager-issuer-prod.yaml`) looks like
+this:
+
+```yaml
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  name: letsencrypt-prod
+spec:
+  acme:
+    # The ACME server URL
+    server: https://acme-v02.api.letsencrypt.org/directory
+    # Email address used for ACME registration
+    email: youremail@goeshere.com
+    # Name of a secret used to store the ACME account private key
+    privateKeySecretRef:
+      name: letsencrypt-prod
+    solvers:
+    # An empty 'selector' means that this solver matches all domains
+    - selector: {}
+      http01:
+        ingress:
+          class: nginx
+```
+
+The only field that is strictly necessary to update is the `email` field.
+
+Once this file is updated and saved, we'll create it via kubectl:
+
+```console
+$ kubectl apply -f cert-manager-issuer-prod.yaml
+```
+
+[ACME Cert Issuer resources]: https://cert-manager.io/docs/configuration/acme/#creating-a-basic-acme-issuer
+
+### Configure Brigade
+
+Now we're ready to add cert manager configuration to the Brigade chart values
+file, `brigade2-values.yaml`.  In fact, there are only a few lines to add under
+the `apiserver.ingress.annotations` section (the same section updated when
+configuring ingress):
+
+```yaml
+apiserver:
+  ingress:
+    enabled: true
+    annotations:
+      kubernetes.io/ingress.class: nginx
+      # The next three lines represent the newly added configuration
+      kubernetes.io/tls-acme: "true"
+      cert-manager.io/cluster-issuer: "letsencrypt-prod"
+      cert-manager.io/acme-challenge-type: http01
+      nginx.ingress.kubernetes.io/secure-backends: "true"
+      nginx.ingress.kubernetes.io/backend-protocol: "HTTPS"
+```
+
+## Third-Party Authentication
+
+The default mode of authentication into Brigade, that of a root user and
+password, is highly insecure and does not scale for a setup involving
```

Comment:
I'm wary of the phrase "highly insecure" because, for some readers, it might not reflect well on us. I wonder if we could improve here by making an analogy to an operating system and stating that the root user, if enabled, has a broad set of permissions that aren't suitable for sharing.